contributors/devel/issues.md

```diff
@@ -1 +1,222 @@
+- `kind/feature`: New functionality.
+- `kind/cleanup`: Adding tests, refactoring, fixing old bugs.
+
+## Bot communication
+
+The bot will communicate the state of an issue in the active milestone via comments and labels.
+
+### Comments
+
+All bot comments will mention issue owners and link to this doc.  Bot
+comments in the workflow section of this document will only include
+the message but be presumed to include the header and footer in the
+following example:
+
+```
+@pwittrock @droot
+
+<message>
+
+Additional instructions available [here](<link to this doc>)
+```
+
+### Labels
+
+The following labels are used by the bot to track the state of an
+issue in the milestone:
+
+ - milestone/incomplete-labels - one or more of the required `kind/`, `priority`/ or `sig/` labels are missing
+ - milestone/needs-approval - the `status/approved-for-milestone` label is missing
+ - milestone/needs-attention - a status label is missing or an update is required
+ - milestone/removed - the issue was removed from the milestone
+
+These labels are mutually exclusive - only one will appear on an issue at once.
+
+## Workflow
+
+1. An issue is added to the current release milestone (either through creation or update)
+  - Bot checks to make sure all required labels are set on the issue
+  - If any labels are missing, the bot comments listing the missing labels and applies the `milestone/incomplete-labels` label.
+    ```
+    **Action required**: Issue is missing the following required labels.  Set the labels or the issue
+    will be moved out of the milestone within 3 days.
+
+    - priority
+    - severity
+    ```
+  - **If required labels are not applied within 3 days of being moved to the milestone, the bot will move the issue out of the milestone and apply the `milestone/removed` label (unless the issue is critical-urgent).**
+  - If the required labels are present, the bot checks whether the issue has the `status/approved-for-milestone` label.
+  - If the approved label is not present, the bot comments indicating that the label must be applied by a SIG maintainer and applies the `milestone/needs-approval` label.
+    ```
+    **Action required**: This issue must have the `status/approved-for-milestone` label applied
+    by a SIG maintainer.
+    ```
+  - If the approved label is present, the bot comments summarizing the label state and removes the other `milestone/*` labels.
+    ```
+    Issue label settings:
+
+    sig/node: Issue will be escalated to SIG node if needed
+    priority/critical-urgent: Never automatically move out of a release milestone.
+                              Escalate to SIG and contributor through all available channels.
+    kind/bug: Fixes a bug.
+    ```
+  - **If the approved label is not applied within 7 days of the `milestone/needs-approval` label being applied, the bot will move the issue out of the milestone and apply the `milestone/removed` label (unless the issue is critical-urgent).**
```

Comment:
I believe it is only 4 days.

Reply:
milestoneOptApprovalGracePeriod is 168*time.Hour or seven days.  "The grace period to wait before removing a non-blocking issue/pr without sig approval from the active milestone."